ci/kind/kind-setup.sh

```diff
@@ -665,6 +666,21 @@ if version_lt "$kind_version" "0.12.0" && [[ "$KUBE_PROXY_MODE" == "none" ]]; th
     exit 1
 fi
 
+# Build Kind node image with the specified K8s version if it doesn't exist in local and remote registry neither
```

Comment:
Remove "neither", or change to "in either the local or remote registry".

Reply:
According to Quan's comment, this part is moved to `conformance.yml`, this comment is removed.